lib/hyrax/specs/shared_specs/hydra_works.rb

```diff
@@ -26,6 +26,63 @@
   it { is_expected.to respond_to :work? }
 end
 
+RSpec.shared_examples 'belongs to collections' do
+  subject(:model)        { described_class.new }
+  let(:adapter)          { Valkyrie::MetadataAdapter.find(:test_adapter) }
+  let(:persister)        { adapter.persister }
+  let(:query_service)    { adapter.query_service }
+  let(:collection_class) { Hyrax::PcdmCollection }
+
+  describe 'collection membership' do
+    it 'is in no collections default' do
+      expect(model.member_of_collection_ids).to be_empty
+    end
+
+    it 'can be added to collections' do
+      collection_ids = [Valkyrie::ID.new('coll_1'), Valkyrie::ID.new('coll_2')]
+
+      expect { model.member_of_collection_ids = collection_ids }
+        .to change { model.member_of_collection_ids }
```

Comment:
Do we need a `.from` assertion as well?

Reply:
it doesn't seem relevant to me, but also would most likely be harmless.

if you make the suggestion i'll commit it.

Reply:
The spec reads well. There is an assumption that change has happened, and I don't quite know how RSpec treats change without the full `.from` and `.to` preamble. I'm okay with it as is.